src/garage/sampler/off_policy_vectorized_sampler.py

```diff
@@ -186,28 +174,20 @@ def obtain_samples(self, itr, batch_size=None, whole_paths=True):
                             ['undiscounted_return'],
                             success_count=running_paths[idx]['success_count']))
 
-                    if isinstance(self.algo.replay_buffer, PathBuffer):
-                        obs_space = self._env_spec.observation_space
-
-                        flattened_obses = obs_space.flatten_n(
-                            np.asarray(running_paths[idx]['observations']))
-                        flattened_next_obses = obs_space.flatten_n(
-                            np.asarray(
-                                running_paths[idx]['next_observations']))
-                        flattened_rewards = np.asarray(
-                            running_paths[idx]['rewards']).reshape(-1, 1)
-                        flattened_terminals = np.asarray(
-                            running_paths[idx]['dones']).reshape(-1, 1)
-                        flattened_actions = self._env_spec.action_space. \
-                            flatten_n(np.asarray(
-                                        running_paths[idx]['actions']))
-
-                        self.algo.replay_buffer.add_path(
-                            dict(rewards=flattened_rewards,
-                                 terminals=flattened_terminals,
-                                 observations=flattened_obses,
-                                 next_observations=flattened_next_obses,
-                                 actions=flattened_actions))
+                    path_dict = {}
+
+                    path_dict['observations'] = obs_space.flatten_n(
+                        running_paths[idx]['observations'])
+                    path_dict['next_observations'] = obs_space.flatten_n(
+                        running_paths[idx]['next_observations'])
+                    path_dict['rewards'] = np.asarray(
+                        running_paths[idx]['rewards']).reshape(-1, 1)
+                    path_dict['terminals'] = np.asarray(
+                        running_paths[idx]['dones']).reshape(-1, 1)
+                    path_dict['actions'] = self._env_spec.action_space. \
```

Comment:
please enclose continuations in () rather than using \ for continuations (PEP8)

Reply:
Fixed this, though I think \ is more readable.